.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
+
+on:
+  pull_request:
+  push:
+    branches: ["master"]
+
+jobs:
+  release:
+    name: Trigger release
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, 'ci skip') && !contains(github.event.head_commit.message, 'skip ci')"
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Prepare repository
+        run: |
+          git fetch --depth=1 origin +refs/tags/*:refs/tags/*
+          git fetch --no-tags --prune --depth=1 origin +refs/heads/*:refs/remotes/origin/*
+          git fetch --prune --unshallow
+
+      - name: Install
+        run: npm ci
+
+      - name: Bundle
+        run: npm run bundle
+
+      - name: Lint
+        run: npm run lint
+
+      - name: Create release
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
These need to be added to the project by an admin right? Or did you already do this?

Reply:
I have added `NPM_TOKEN` already (you can see it in the `secrets` section of the repository settings) and `GITHUB_TOKEN` is automatically generated for a short period of time for each GitHub action job so no need to take care of it.